src/main/java/com/google/api/generator/engine/writer/JavaWriterVisitor.java

```diff
@@ -139,6 +140,27 @@ public void visit(AnnotationNode annotation) {
     newline();
   }
 
+  @Override
+  public void visit(NewObjectExpr newObjectValue) {
```

Comment:
nit: I suggest name the variable as `newObjectExpr`

Reply:
+1